ansible/roles/kong-api/defaults/main.yml

```diff
@@ -5193,3 +5193,75 @@ kong_apis:
       config.limit_by: credential
     - name: request-size-limiting
       config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+- name: programBulkJobRequestCreate
```

Comment:
syntax issues are present in this yaml @venkateshwarans please check all the below apis as well, they have same issues.